app/javascript/components/forms/profile/component.jsx

```diff
@@ -78,47 +81,74 @@ class ProfileForm extends PureComponent {
                         validate={[validateEmail]}
                         required
                       />
-                      <Select
-                        name="language"
-                        label="language"
-                        options={getLanguages()}
-                        placeholder="Select a language"
-                        required
-                      />
                       <Select
                         name="sector"
                         label="sector"
-                        options={sectors.map(s => ({ label: s, value: s }))}
-                        placeholder="Select a language"
-                        required
+                        options={Object.keys(sectors).map(s => ({
+                          label: s,
+                          value: s
+                        }))}
+                        placeholder="Select a sector"
                       />
+                      {values.sector &&
+                        sectors[values.sector] && (
+                        <Radio
+                          name="subsector"
+                          label="Role"
+                          options={sectors[values.sector].map(s => ({
+                            label: s,
+                            value: s.replace(/( )+|(\/)+/g, '_')
```

Comment:
Not sure this is totally necessary for this native select.

Reply:
In fact maybe we should refactor it to only accept arrays or string and scrap the objects all together.

Reply:
yeah I agree we should do this in the component, as it's repeated a couple of times, but I wasn't sure if we were gonna keep that after the redesign

Reply:
Lets update it then. Redesign?